squidb/src/com/yahoo/squidb/sql/Function.java

```diff
@@ -129,10 +129,10 @@ protected String separator() {
     }
 
     /**
-     * Create a Function that counts all rows
+     * Create a Function that counts all rows (i.e. count(*))
      */
     public static Function<Integer> count() {
-        return new ArgumentFunction<Integer>("COUNT", 1);
+        return new RawFunction<Integer>("COUNT(*)");
```

Comment:
Interesting. For some reason i thought the count(1) was advantageous over count(*), but can't remember why...